articles/aks/howto-deploy-java-liberty-app.md

```diff
@@ -204,9 +227,6 @@ To avoid Azure charges, you should clean up unneeded resources.  When the cluste
 az group delete --name java-liberty-project --yes --no-wait
 ```
 
-> [!NOTE]
-> When you delete the cluster, the Azure Active Directory service principal used by the AKS cluster is not removed. For steps on how to remove the service principal, see [AKS service principal considerations and deletion](kubernetes-service-principal.md#additional-considerations). If you used a managed identity, the identity is managed by the platform and does not require removal.
-
```

Comment:
> I wasn’t able to work out how to do this deletion based on the docs.  I got the error:
AADSTS50076: Due to a configuration change made by your administrator, or because you moved to a new location, you must use multi-factor authentication to access '797f4846-ba00-4fd7-ba43-dac1f8f63013'.
Trace ID: bee706c6-746a-4cc7-9994-ca4a51430b00
Correlation ID: d7b70873-b4c1-41eb-ac2a-c8d96e2a0ae8
Timestamp: 2021-01-22 14:36:14Z
az ad sp delete: error: argument --id: expected one argument

I can successfully delete the service principal by running:
```
az ad sp delete --id <service-principal-client-id>
```
So I suspect the issue you had may relate to the free trial account. Anyway, as it's not a mandatory step to complete this guide, I removed this note to avoid confusion.

Reply:
I couldn't even get hold of the service principal client id to know what to delete.  The suggested command:
```
az aks show -g java-liberty-project -n myAKSCluster --query servicePrincipalProfile.clientId -o tsv
```
Did not work.

However, given at this point my cluster had already been deleted, maybe that's the reason why? :/

I don't have time to dig more.